ffi/diplomat/ffi_coverage/tests/missing_apis.txt

```diff
@@ -1,12 +1,4 @@
-icu::properties::script::ScriptExtensionsSet#Struct
-icu::properties::script::ScriptExtensionsSet::contains#FnInStruct
 icu::properties::script::ScriptExtensionsSet::iter#FnInStruct
```

Comment:
@sffc With these changes the only remaining bit is this function. I could create an iterator type, or I could just expose a `/len()` and `.script_at()`, thoughts?

I think the iterator is less nice over FFI until we can have good iterators; but we may potentially change the impl to not be a vector in the future? Though we can just declare that iteration order is unstable.

Reply:
The annoying thing over exposing len/at is that then the Rust types need to do that, but there is no reason to have this on the Rust side (perhaps I can just doc(hidden) them)

Reply:
len/at seems fine. We should use that paradigm more often over FFI.

I see pros and cons to making them public vs hidden in Rust; either is fine.